### ISSUE TITLE
Problem with Kubernetes resource name generation

### ISSUE THREAD

**Issue by amarchese96** [tliron/turandot#15]
I tried to compile the bookinfo example and generate the Kubernetes YAML configuration files using the scriptlet at https://raw.githubusercontent.com/tliron/turandot/main/assets/tosca/profiles/kubernetes/1.0/js/resources/get.js. 
The generated resources names are postfixed with the corresponding capability names (for example in the case of a deployment, the generated name is ${nodeTemplateName}-deployment). 
I noticed that the postfix can be overidden by setting the capability property "metadataNamePostfix", but it seems that this property is not defined in the Kubernetes profile.

**Comment by tliron** [comment]
You are correct! I've recently switched to a system in which the Kubernetes profile is generated automatically, and I think I forgot about this property. Expect a fix soon.
